server/modules/storage/s3/storage.js

```diff
@@ -1,23 +1,76 @@
+const S3 = require('aws-sdk/clients/s3')
+
+/* global WIKI */
+
+/**
+ * AWS S3 Client
+ */
+let s3 = null
+
+/**
+ * Get file extension based on content type
+ */
+const getFileExtension = contentType => {
```

Comment:
i notice that this gets duplicated in other places, i.e., in the `git` and `disk` storage modules. any plans to refactor them?

Reply:
Indeed, I'll make it available as part of the page object.

Reply:
let me do that as part of this PR, unless you have already done so?